file-formats/cfdf/type/zif_aff_cfdf_v1.intf.abap

```diff
@@ -0,0 +1,589 @@
+
+  TYPES:
+    "! <p class="shorttext">Value Help Binding Details</p>
+    "! Value help binding details
+    BEGIN OF ty_gs_value_help_binding,
+      "! <p class="shorttext">Extended CDS View Name</p>
+      "! Extended CDS view name
+      "! $required
+      "! $maxLength 30
+      cds_view_name                TYPE  zif_aff_types_v1=>ty_object_name_30,
+
+      "! <p class="shorttext">Extended CDS View Field Name</p>
+      "! Extended CDS view field name
+      "! $required
+      "! $maxLength 30
+      extended_cds_view_field_name TYPE ty_field_name,
+
+      "! <p class="shorttext">Value Help Field Name</p>
+      "! Value help field name
+      "! $required
+      "! $maxLength 30
+      value_help_field_name        TYPE  ty_field_name,
+    END OF ty_gs_value_help_binding.
+  TYPES ty_gt_value_help_binding TYPE STANDARD TABLE OF ty_gs_value_help_binding
+                                 WITH KEY cds_view_name value_help_field_name.
+
+  TYPES:
+    "! <p class="shorttext">Data Protection and Privacy</p>
+    "! Data protection and privacy
+    BEGIN OF ty_gs_dpp_properties,
+      "! <p class="shorttext">Data Subject</p>
+      "! Data subject
+      "! $maxLength 10
+      "! $values {@link zif_aff_cfdf_v1.data:co_data_subject_id_type}
+      "! $default {@link zif_aff_cfdf_v1.data:co_data_subject_id_type.not_applicable}
+      data_subject_id_type       TYPE ty_data_subject_id_type,
+
+      "! <p class="shorttext">BAdI Implementation for Data Subject</p>
+      "! BAdI implementation for data subject
+      "! $maxLength 30
+      badi_impl_for_data_subject TYPE zif_aff_types_v1=>ty_object_name_30,
+    END OF ty_gs_dpp_properties.
+
+  TYPES:
+    "! <p class="shorttext">Characteristic Reference</p>
+    "! Characteristic reference
+    BEGIN OF ty_gs_characteristic_reference,
+      "! <p class="shorttext">Characteristic Name</p>
+      "! Characteristic name
+      "! $maxLength 30
+      characteristic_name TYPE c LENGTH 30,
+
+      "! <p class="shorttext">Class Name</p>
+      "! Class name
+      "! $maxLength 3
+      class_type          TYPE c LENGTH 3,
+
+      "! <p class="shorttext">Object Type</p>
+      "! Object type
+      "! $maxLength 30
+      object_type         TYPE c LENGTH 30,
+    END OF ty_gs_characteristic_reference.
+
+  TYPES:
+    "! <p class="shorttext">Custom Field</p>
+    "! Custom field
+    BEGIN OF ty_main,
+      "! $required
+      format_version              TYPE zif_aff_types_v1=>ty_format_version,
+
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header                      TYPE zif_aff_types_v1=>ty_header_60_no_abap_lv,
+
+      "! <p class="shorttext">External Name</p>
+      "! External name
+      "! $required
+      "! $maxLength 30
+      external_name               TYPE ty_field_name,
+
+      "! <p class="shorttext">Type</p>
+      "! Type
+      "! $required
+      "! $maxLength 10
```

Comment:
Again, it is wrong here.

Reply:
All occurrences of $maxLength were removed...